MovieMVP/MovieMVP/DetailMoviesViewController/View/DetailMoviesViewController.swift

```diff
@@ -181,10 +202,26 @@ extension SecondViewController: UICollectionViewDataSource, UICollectionViewDele
                 for: indexPath
             ) as? ActorCollectionViewCell
         else { return UICollectionViewCell() }
-        let actor = actor.cellForRowAt(indexPath: indexPath)
-        cell.setCellWithValues(actor)
+        guard let actor = presenter?.actors[indexPath.row] else { return UICollectionViewCell() }
+        cell.setCellWithValues(actor: actor)
         cell.layer.cornerRadius = 5
         cell.clipsToBounds = true
         return cell
     }
 }
+
+// MARK: - DetailMoviesViewable
+
+extension DetailMoviesViewController: DetailMoviesViewable {
+    func succes() {
+        collectionView.reloadData()
+    }
+
+    func failure(_ error: Error) {
+        print(error.localizedDescription)
```

Comment:
```suggestion
        print(error.localizedDescription)
```
Лучше выводить ошибку дерезе Алерт

/// Расширение для UIViewController(Alert)
private enum Constant {
    static let okTextString = "OK"
}

extension UIViewController {
    typealias Closure = (() -> ())?
    func showAlert(title: String?, message: String, handler: Closure) {
        let alertController = UIAlertController(title: title, message: message, preferredStyle: .alert)
        let alertControllerAction = UIAlertAction(title: Constant.okTextString, style: .default) { _ in
            handler?()
        }
        alertController.addAction(alertControllerAction)
        present(alertController, animated: true)
    }
}

Добавь расширение это